pages/orderSuccess.js

```diff
@@ -215,7 +216,10 @@ class OrderSuccessPage extends React.Component {
             <MessageBox type="warning" my={4} mx={2}>
               <FormattedMessage
                 id="collective.user.orderProcessing.manual"
-                defaultMessage="Your donation is pending. Please follow the instructions in the confirmation email to manually pay the host of the collective."
+                defaultMessage="<i18n-bold>Your donation is pending.</i18n-bold> Please follow the instructions in the confirmation email to manually pay the host of the collective."
+                values={{
+                  'i18n-bold': I18nBold,
+                }}
```

Comment:
This is good :+1: Just suggesting an alternative in case you're now aware of it yet (not a requested change)

```suggestion
                defaultMessage="<strong>Your donation is pending.</strong> Please follow the instructions in the confirmation email to manually pay the host of the collective."
                values={I18nFormatters}
```